app/templates/gulp/gulp/config.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var fs = require( 'fs' );
+
+/**
+ * Set up file paths
+ */
+var loc = {
+  src:  './src',
+  dist: './dist',
+  lib:  JSON.parse( fs.readFileSync( './.bowerrc' ) ).directory, // eslint-disable-line no-sync, no-inline-comments, max-len
+  test: './test'
+};
+
+module.exports = {
+  pkg:    JSON.parse( fs.readFileSync( 'bower.json' ) ), // eslint-disable-line no-sync, no-inline-comments, max-len
+  banner:
+      '/*!\n' +
+      ' *  <%%= pkg.name %> - v<%%= pkg.version %>\n' +
+      ' *  <%%= pkg.homepage %>\n' +
+      ' *  Licensed <%%= pkg.license %> by <%= pkg.author.name %> <%= pkg.author.email %>\n' +
+      ' */',
+  lint: {
+    src: [
+      loc.src + '/static/js/**/*.js',
+      loc.test + '/unit_tests/**/*.js',
+      loc.test + '/browser_tests/**/*.js'
+    ],
+    gulp: [
+      'gulpfile.js',
+      'gulp/**/*.js'
+    ]
+  },
+  test: {
+    src:   loc.src + '/static/js/**/*.js',
+    tests: loc.test
+  },
+  clean: {
+    dest: loc.dist
+  },
+  styles: {
+    cwd:      loc.src + '/static/css',
+    src:      '/main.less',
+    dest:     loc.dist + '/static/css',
+    settings: {
+      paths: [
+        loc.lib,
+        loc.lib + '/cf-typography/src'
+      ],
+      compress: true
+    }
+  },
+  scripts: {
+    src: [
+      loc.lib + '/jquery/dist/jquery.js',
+      loc.lib + '/jquery.easing/js/jquery.easing.js',
+      loc.lib + '/cf-*/src/js/*.js',
+      loc.src + '/static/js/app.js'
+    ],
+    dest: loc.dist + '/static/js/',
+    name: 'main.js'
+  },
+  images: {
+    src:  loc.src + '/static/img/**',
+    dest: loc.dist + '/static/img'
+  },
+  copy: {
+    files: {
+      src: [
+        loc.src + '/**/*.html',
+        loc.src + '/**/*.pdf',
+        loc.src + '/_*/**/*',
+        loc.src + '/robots.txt',
+        loc.src + '/favicon.ico',
+        '!' + loc.lib + '/**/*.html'
+      ],
+      dest: loc.dist
+    },
+    icons: {
+      src:  loc.lib + '/cf-icons/src/fonts/*',
+      dest: loc.dist + '/static/fonts/'
+    },
+    vendorjs: {
+      src: [
+        loc.lib + '/box-sizing-polyfill/boxsizing.htc',
```

Comment:
This is only needed if cf-grid is included. Just flagging if that for some reason wouldn't be needed.